model/utils.go

```diff
@@ -20,6 +20,13 @@ import (
 	"github.com/pborman/uuid"
 )
 
+const (
+	LOWERCASE_LETTERS = "abcdefghijklmnopqrstuvwxyz"
+	UPPERCASE_LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	NUMBERS           = "0123456789"
+	SYMBOLS           = " !\"#$%&'()*+,-./:;<=>?@[]^_`|~"
```

Comment:
The triple backslash is actually because I was escaping the backslash as well as the closing square bracket. `/[\\\]]/` matches `\` or `]`